aspnetcore/client-side/bower.md

```diff
@@ -14,231 +14,108 @@ uid: client-side/bower
-<environment names="Staging,Production">
-    <link rel="stylesheet" href="https://ajax.aspnetcdn.com/ajax/bootstrap/3.3.6/css/bootstrap.min.css"
-        asp-fallback-href="~/lib/bootstrap/dist/css/bootstrap.min.css"
-        asp-fallback-test-class="sr-only" asp-fallback-test-property="position" asp-fallback-test-value="absolute" />
-    <link rel="stylesheet" href="~/css/site.min.css" asp-append-version="true" />
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css"
-        asp-fallback-href="~/lib/font-awesome/css/font-awesome.min.css" />
-</environment>
-```
+Replace the contents of the *Views\Home\About.cshtml* Razor file with the following markup:
 
-Open `Views\Home\Index.cshtml` and add the icons to the **How to**, **Overview**, and **Run & Deploy** headers. You'll notice when typing, IntelliSense is available on the css classes.
-
-![css intellisense in cshtml](bower/_static/css-intellisense.png)
-
-Add the question icon to the **How to** header.
-
-```html
-<div class="col-md-3">
-    <h2><i class="fa fa-question"></i> How to</h2>
-    <ul>
-        <li><a href="http://go.microsoft.com/fwlink/?LinkID=398600">Add a Controller and View</a></li>
-        <li><a href="http://go.microsoft.com/fwlink/?LinkID=699562">Add an appsetting in config and access it in app.</a></li>
-        <li><a href="http://go.microsoft.com/fwlink/?LinkId=699315">Manage User Secrets using Secret Manager.</a></li>
-        <li><a href="http://go.microsoft.com/fwlink/?LinkId=699316">Use logging to log a message.</a></li>
-        <li><a href="http://go.microsoft.com/fwlink/?LinkId=699317">Add packages using NuGet.</a></li>
-        <li><a href="http://go.microsoft.com/fwlink/?LinkId=699318">Add client packages using Bower.</a></li>
-        <li><a href="http://go.microsoft.com/fwlink/?LinkId=699319">Target development, staging or production environment.</a></li>
-    </ul>
-</div>
-```
+[!code-html[Main](bower\sample\About.cshtml)]
 
-Run the application to see the changes.
+Run the app and navigate to the About view to verify the font-awesome package works.
 
-![app with icons after changes](bower/_static/app-with-icons.png)
+![show home/lib/app/setting icons](bower/_static/version-intellisense.png)
 
 ## Exploring the client build process
 
-Most ASP.NET Core project templates are already configured to use Bower. This next walkthrough starts with an empty ASP.NET Core project and adds each piece manually, so you can get a feel for how Bower is used in a project. See what happens to the project structure and the runtime output as each configuration change is made to the project.
+Most ASP.NET Core project templates are already configured to use Bower. This next walkthrough starts with an empty ASP.NET Core project and adds each piece manually, so you can get a feel for how Bower is used in a project. You can what happens to the project structure and the runtime output as each configuration change is made.
```

Comment:
I think the word "see" is missing here. Should this sentence begin with "You can see"?